TombEngine/Objects/TR3/Vehicles/big_gun.cpp

```diff
@@ -111,36 +111,30 @@ namespace TEN::Entities::Vehicles
 
 	void BigGunFire(ItemInfo* bigGunItem, ItemInfo* laraItem)
 	{
-		auto* lara = GetLaraInfo(laraItem);
 		auto* bigGun = GetBigGunInfo(bigGunItem);
 
 		short itemNumber = CreateItem();
-		auto* projectileItem = &g_Level.Items[itemNumber];
-
 		if (itemNumber != NO_ITEM)
```

Comment:
You can invert the condition and do an early exit from the function, this will simplify code a bit. Since your changes are minor, I don't think it will affect PR readability that much.

Reply:
Done